command/agent.go

```diff
@@ -507,25 +508,40 @@ func (c *AgentCommand) Run(args []string) int {
 				ps, err := cacheboltdb.NewBoltStorage(&cacheboltdb.BoltStorageConfig{
 					Path:   config.Cache.Persist.Path,
 					Logger: cacheLogger.Named("cacheboltdb"),
-					AAD:    aad,
 				})
 				if err != nil {
 					c.UI.Error(fmt.Sprintf("Error opening persistent cache: %v", err))
 					return 1
 				}
+
 				// Get the token from bolt for retrieving the encryption key,
 				// then setup encryption so that restore is possible
-				token, err := ps.GetRetrievalToken()
+				key, err := ps.GetRetrievalToken()
```

Comment:
Just noticing that we we're using `key` here with GetRetrievalToken(), but using `token` down on line 610 with StoreRetrievalToken(): https://github.com/hashicorp/vault/blob/673cf14ac7ce30ea5543403d9d6119b6ac2f385f/command/agent.go#L609-L618
We may want to use the same term both places?